src/drivers/imx/sdma.c

```diff
@@ -942,10 +942,10 @@ static int sdma_get_data_size(struct dma_chan_data *channel, uint32_t *avail,
 	*avail = *free = 0;
 	switch (channel->direction) {
 	case DMA_DIR_MEM_TO_DEV:
-		*avail = result_data;
+		*free = result_data;
 		break;
 	case DMA_DIR_DEV_TO_MEM:
-		*free = result_data;
+		*avail = result_data;
```

Comment:
Typo (released SHIFT key too soon?) in commit message for this change.

Reply:
You mean DEv instead of DEV?